csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/EnvironmentVariableNames.cs

```diff
@@ -25,7 +25,17 @@ internal class EnvironmentVariableNames
         /// <summary>
         /// Specifies the NuGet feeds to exclude from the responsiveness check.
         /// </summary>
-        public const string ExcludedNugetFeedsFromResponsivenessCheck = "CODEQL_EXTRACTOR_CSHARP_BUILDLESS_NUGET_FEEDS_EXCLUDED_FROM_CHECK";
+        public const string ExcludedNugetFeedsFromResponsivenessCheck = "CODEQL_EXTRACTOR_CSHARP_BUILDLESS_NUGET_FEEDS_CHECK_EXCLUDED";
+
+        /// <summary>
+        /// Specifies the timeout for the initial check of NuGet feeds responsiveness.
```

Comment:
Maybe add some more description on the expected format and type of the content of the environment variable ."Specifies the timeout (as an integer) in milliseconds..". Same comment applies to the other environment variables (now that is has started to be more complicated and we have lots of variables controlling the behaviour).